Python/ceval.c

```diff
@@ -4517,13 +4518,8 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             PyObject *cls = TOP();
             PyTypeObject *cls_type = Py_TYPE(cls);
             assert(cls_type->tp_dictoffset > 0);
```

Comment:
You can now wrap these two lines in `#ifdef Py_DEBUG` to get rid of the unused variable warning :)